api/v1alpha2/zz_generated.conversion.go

```diff
@@ -617,6 +617,7 @@ func autoConvert_v1alpha3_AWSMachineSpec_To_v1alpha2_AWSMachineSpec(in *v1alpha3
 	out.SSHKeyName = in.SSHKeyName
 	out.RootDeviceSize = in.RootDeviceSize
 	out.NetworkInterfaces = *(*[]string)(unsafe.Pointer(&in.NetworkInterfaces))
+	// WARNING: in.CloudInit requires manual conversion: does not exist in peer-type
```

Comment:
Tackling this later?

Reply:
Just discussed with @ncdc as to the sequence of doing these things since we'll also be backporting it. Will be tackled in this PR shortly.